Tests/Microsoft.AppCenter.Crashes.Test.Windows/CrashesTest.cs

```diff
@@ -102,44 +108,95 @@ public void ApplyEnabledStateCleansUp()
         }
 
         [TestMethod]
-        public void OnChannelGroupReadySendsPendingCrashes()
+        public void OnChannelGroupReadySendsAllPendingCrashesIfShouldProcessNotImplemented()
         {
-            var mockFile1 = Mock.Of<File>();
-            var mockFile2 = Mock.Of<File>();
+            var mockErrorLogFile1 = Mock.Of<File>();
+            var mockErrorLogFile2 = Mock.Of<File>();
+            var mockExceptionFile1 = Mock.Of<File>();
+            var mockExceptionFile2 = Mock.Of<File>();
             var mockErrorLogHelper = Mock.Of<ErrorLogHelper>();
             ErrorLogHelper.Instance = mockErrorLogHelper;
-            var expectedprocessId = 123;
+            var expectedProcessId = 123;
             var expectedManagedErrorLog1 = new ManagedErrorLog
             {
                 Id = Guid.NewGuid(),
                 Timestamp = DateTime.Now,
                 AppLaunchTimestamp = DateTime.Now,
                 Device = new Microsoft.AppCenter.Ingestion.Models.Device(),
-                ProcessId = expectedprocessId
+                ProcessId = expectedProcessId
             };
             var expectedManagedErrorLog2 = new ManagedErrorLog
             {
                 Id = Guid.NewGuid(),
                 Timestamp = DateTime.Now,
                 AppLaunchTimestamp = DateTime.Now,
                 Device = new Microsoft.AppCenter.Ingestion.Models.Device(),
-                ProcessId = expectedprocessId
+                ProcessId = expectedProcessId
             };
 
             // Stub get/read/delete error files.
-            Mock.Get(ErrorLogHelper.Instance).Setup(instance => instance.InstanceGetErrorLogFiles()).Returns(new List<File> { mockFile1, mockFile2 });
-            Mock.Get(ErrorLogHelper.Instance).Setup(instance => instance.InstanceReadErrorLogFile(mockFile1)).Returns(expectedManagedErrorLog1);
-            Mock.Get(ErrorLogHelper.Instance).Setup(instance => instance.InstanceReadErrorLogFile(mockFile2)).Returns(expectedManagedErrorLog2);
+            Mock.Get(ErrorLogHelper.Instance).Setup(instance => instance.InstanceGetErrorLogFiles()).Returns(new List<File> { mockErrorLogFile1, mockErrorLogFile2 });
```

Comment:
If we could find a way to factor out the mocking logic for the saved errors logs and saved exception logic that would be nice.  I don't think it's required if the function would be too convoluted, though

Reply:
When there are 2 files in a test it's specific to a single test.
In unit tests, some amount of duplication is ok, factoring can make tests harder to read and not always easy to maintain.

Reply:
There are other tests that could benefit some factoring, I improved those instead.